harmony_netcdf_to_zarr/mosaic_utilities.py

```diff
@@ -58,10 +59,13 @@ class DimensionInformation:
 
     """
     def __init__(self, dimension_path: str, dimension_values: np.ndarray,
-                 dimension_units: str):
+                 dimension_units: str, bounds_path: str = None,
+                 bounds_values: np.ndarray = None):
         self.dimension_path = dimension_path
         self.values = dimension_values
         self.units = dimension_units
+        self.bounds_path = bounds_path
+        self.bounds_values = bounds_values
```

Comment:
The bounds information, as stored in the output `DimensionInformation` instances will be used when writing the output Zarr store. The aggregated `bounds` variables can be written at the same time as the aggregated dimension variables (probably before iterating through the input granules).